src/components/form/form.ts

```diff
@@ -69,7 +69,7 @@ export class MForm extends ModulVue {
     private handleErrors(): void {
         this.emit(MFormEvents.formError, {
             form: this.form,
-            totalNbOfErrors: this.form.totalNbOfErrors,
+            totalNbOfErrors: this.form.nbFieldsThatHasError,
```

Comment:
Préférence personelle mais je préfère utilisé total ou count plutôt que nb dans les noms de variable.

ici `totalErrors: this.form.ErrorFieldsCount`